UniversalAlgebra/GaloisConnections/Basic.lagda

```diff
@@ -95,8 +95,25 @@ module _ {ℓ ρ : Level}{𝒜 ℬ : Type ℓ} where
  →←Closed {B = B}{R} = (R ⃖ B) ⃗ R ⊆ B
 
 
+module _ {α β ρ : Level}{𝒜 : Type α}{ℬ : Type β}{R : REL 𝒜 ℬ ρ} where
 
+ Foo : Pred 𝒜 (α ⊔ β ⊔ ρ) → Pred ℬ (α ⊔ β ⊔ ρ)
```

Comment:
Foo and Bar are placeholder names in CS that are meant to be replaced by something meaningful...  They were never real suggestions!